### PR TITLE
Switch tools management to bingo

### PR DESCRIPTION
**Pull request by rashmigottipati**
All the files in .bingo are auto-generated

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -153,18 +149,16 @@ test-e2e-setup:: build dev-install cluster-create
...
 cluster-create::
-	[[ "`$(TOOLS_DIR)/kind get clusters`" =~ "$(KIND_CLUSTER)" ]] || $(TOOLS_DIR)/kind create cluster --image="kindest/node:v$(K8S_VERSION)" --name $(KIND_CLUSTER)
-	$(TOOLS_DIR)/kind export kubeconfig --name $(KIND_CLUSTER)
+	[[ "`$(KIND) get clusters`" =~ "$(KIND_CLUSTER)" ]] || $(KIND) create cluster --image="kindest/node:v$(K8S_VERSION)" --name $(KIND_CLUSTER)
+	$(KIND) export kubeconfig --name $(KIND_CLUSTER)
 
 .PHONY: dev-install
 dev-install::
-	$(SCRIPTS_DIR)/fetch kind 0.20.0
 	$(SCRIPTS_DIR)/fetch kubectl $(K8S_VERSION) # Install kubectl AFTER envtest because envtest includes its own kubectl binary
```
Comment (grokspawn) [inline Makefile:157]: should kubectl also be bingo'd?

File: tools/scripts/fetch [hunk 2]
```diff
@@ -22,10 +22,6 @@ fetch() {
       ver_cmd="${DEST}/golangci-lint --version 2>/dev/null | cut -d' ' -f4"
```
Comment (grokspawn) [inline tools/scripts/fetch:22]: should golangci-lint be bingo'd?

**Review by everettraven — APPROVED** [review]
Looks good. Could probably remove https://github.com/operator-framework/ansible-operator-plugins/pull/59/files#diff-2d2809b5329aefe0d12b6ec9a166d429b1089b1b7f8577e7cce27846abd53cbcR29-R34 but IMO shouldn't block this.

File: tools/scripts/fetch (hunk 2 above)
Comment (rashmigottipati) [reply tools/scripts/fetch]: updated in the latest commit.

File: Makefile (hunk 1 above)
Comment (rashmigottipati) [reply Makefile:157]: it could be, but I couldn't do it as `bingo get k8s.io/kubectl@latest` didn't work for me, so I left it intact.

**Comment by rashmigottipati** [comment]
@everettraven the link you referenced in your comment points to the entire diff, were you referring to anything specific? I am assuming you're referring to removing `goreleaser` from the `fetch` script. If so, I fixed it in the latest commit. 

Let me know if I'm missing anything else.

**Pull request merged**